hades/templates/reset_password.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block title %}
+    <title>Reset Password</title>
+{% endblock %}
+
+{% block content %}
+    <h1 class="h3 mb-3 font-weight-normal">Please enter your new desired password</h1>
+    <label for="inputUsername" class="sr-only">Username</label>
+    <input type="text" id="inputUsername" class="form-control" placeholder="Username" name="username"
+           required>
+    <br>
+    <label for="inputNewPassword" class="sr-only">New Password</label>
+    <input type="password" id="inputNewPassword" class="form-control" placeholder="New password" name="new_password"
+           required>
+    <!--
+    TODO: Add JS validation to ensure password has been entered again
```

Comment:
Do you mean to check if old password is used or to check if another password has been inputted on the same page?

Reply:
I meant just to verify password, most change password pages make enter it twice for confirmation